nixos/modules/services/web-apps/filebrowser.nix

```diff
@@ -0,0 +1,168 @@
+{
+  config,
+  pkgs,
+  lib,
+  utils,
+  ...
+}:
+let
+  cfg = config.services.filebrowser;
+  inherit (lib) types;
+  format = pkgs.formats.json { };
+in
+{
+  options = {
+    services.filebrowser = {
+      enable = lib.mkEnableOption "FileBrowser";
+
+      package = lib.mkPackageOption pkgs "filebrowser" { };
+
+      openFirewall = lib.mkOption {
+        default = false;
+        description = ''
+          Whether to automatically open the ports for FileBrowser in the firewall.
+        '';
+        type = types.bool;
+      };
+
+      stateDir = lib.mkOption {
+        default = "filebrowser";
+        description = ''
+          The directory below `/var/lib` where FileBrowser stores its state.
+        '';
+        type = types.str;
+      };
+
+      cacheDir = lib.mkOption {
+        default = "filebrowser";
+        description = ''
+          The directory below `/var/cache` where FileBrowser stores its cache.
+        '';
+        type = types.nullOr types.str;
+      };
```

Comment:
I dont think we should make them configurable. What's the benefits of that?

Reply:
Yeah, on second thought, I agree. I just removed those options.